pages/v4/Library-Usage.md

```diff
@@ -23,19 +23,19 @@ There are various kinds of libraries that you can use with Duality, some of them
 
 ## Duality Plugins
 
-In the easiest case, you simply want your game plugin to reference another Duality plugin. Just use the "Add Reference" dialog and browse into the Plugins folder of your Duality project to select the plugin you need, and that's it. No additional steps required.
+In the easiest case, you simply want your game plugin to reference another Duality plugin. Just use the "Add Reference" dialog and browse to the dll or project file of the plugin, and that's it. No additional steps required.
 
 However, you should be careful not to mix core and editor plugins. While an editor plugin can reference whatever it wants, a core plugin should never rely on an editor plugin - otherwise, your game would depend on the presence of an editor, rendering the launcher application non-functional.
 
 ## Managed Libraries
 
-You can reference managed libraries that are not Duality plugins as well, same procedure as above. In order to make sure that the Duality runtime is able to locate your referenced library, put it into the Plugins folder, right next to your game plugin. In most cases, that's all you need to know.
+You can reference managed libraries that are not Duality plugins as well, same procedure as above. In most cases, that's all you need to know.
 
-### Portability and (non-) Portable Class Libraries
+### Portability and (non-) .NET Standard Class Libraries
 
-By default, Duality game plugins are [Portable Class Libraries](Framework-Structure.md), which means that they limit themselves to a subset of the .Net Framework that is known to be portable across a range of platforms. To keep this contract intact, you can only reference libraries that use a similar or smaller subset of .Net, thus retaining portability.
+By default, Duality game plugins are [.NET Standard Class Libraries](Framework-Structure.md), which means that they limit themselves to a subset of .NET that is known to be portable across all .NET implementations. To keep this contract intact, you can only reference libraries that use a similar or smaller subset of .NET, thus retaining portability. See the [microsoft .NET Standard docs page](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) for more info about .NET Standard.  
```

Comment:
Would it be necessary to specify that is .NET Standard 2.0?

Reply:
No, not in this article but in https://docs.duality2d.net/pages/v4/Framework-Structure.html

Reply:
There is already a link to that article and repeating these details means we have to keep them in sync as well